### PR TITLE
Assignment and Student Works exist and show in front end (KHO-150)

### PR DESCRIPTION
**Pull request by alexangione419**
# Checklist

- [ ] I have performed a self-review of my code
- [ ] I have reached out to another developer to review my code
- [ ] New and existing unit tests pass locally with my changes
- [ ] All Lint and CI checks are passing

### CODE REVIEW

File: backend/internal/handlers/classrooms/assignments/assignments.go [hunk 1]
```diff
@@ -1,29 +1,68 @@
...
-		return c.SendStatus(fiber.StatusNotImplemented)
+		var assignmentData models.AssignmentOutline
+		error := c.BodyParser(&assignmentData)
+		if error != nil {
+			return errs.InvalidRequestBody(models.AssignmentOutline{})
+		}
+
+		createdAssignment, err := s.store.CreateAssignment(c.Context(), assignmentData)
+		if err != nil {
+			return err
```
Comment (CamPlume1) [inline backend/internal/handlers/classrooms/assignments/assignments.go:60]: Use errors package please

File: backend/internal/handlers/classrooms/assignments/assignments.go [hunk 2]
```diff
@@ -1,29 +1,68 @@
...
-		// Implement logic here
-		return c.SendStatus(fiber.StatusNotImplemented)
+		assignmentID, err := strconv.ParseInt(c.Params("assignment_id"), 10, 64)
+		if err != nil {
+			return errs.BadRequest(err)
+		}
+
+		assignment, err := s.store.GetAssignmentByID(c.Context(), assignmentID)
+		if err != nil {
+			return err
```
Comment (CamPlume1) [inline backend/internal/handlers/classrooms/assignments/assignments.go:41]: Use errs package

File: backend/internal/handlers/classrooms/assignments/assignments.go [hunk 3]
```diff
@@ -1,29 +1,68 @@
...
-		// Implement logic here
-		return c.SendStatus(fiber.StatusNotImplemented)
+		classroomID, err := strconv.ParseInt(c.Params("classroom_id"), 10, 64)
+		if err != nil {
+			return errs.BadRequest(err)
+		}
+
+		assignments, err := s.store.GetAssignmentsInClassroom(c.Context(), classroomID)
+		if err != nil {
+			return err
```
Comment (CamPlume1) [inline backend/internal/handlers/classrooms/assignments/assignments.go:22]: Use errs packages. Feel free to create a custom error for these cases

**Review by CamPlume1 — CHANGES_REQUESTED** [review]
Looks generally awesome, but please incorporate errs package in service layer

**Review by CamPlume1 — APPROVED** [review]
Looks good

**Review by CamPlume1 — APPROVED** [review]
lgtm

**Review by ntietje1 — APPROVED** [review]
lgtm lfg

**Pull request merged**